### DISCUSSION TITLE
Hound

### DISCUSSION THREAD

**Discussion by LouderthanThunder** [Lambosaurus/hives-and-torches#44] [Feats]
You gain a trusty good boi 

His stats are 1/0/1/0

Bite attack

Can give flank

Can give moral bonus 

Gives moral debuff when killed

**Comment by Lambosaurus** [comment]
I like flank and a bite. Or even a grapple attack.
We don't yet have morale rules. I don't think pcs should be forced to roll morale.

**Reply by Lambosaurus** [reply]
I think the problem is this will give you too much for one feat. But if you make the hound too weak, it will just die all the time.
Perhaps it needs to be broken into 2 feats? A second feat to improve its fatigue pool and give it grapple?

**Reply by LouderthanThunder** [reply]
Yup that's fine. Or multiple feats. Kinda posted it to see how y'all feel about summons in general

**Reply by Lambosaurus** [reply]
Minions will be very strong, so I think we need to be very careful how we hand them out.
I really want to play a houndmaster tho, so I'll allow it.
